### ISSUE TITLE
Can Install, Cant Enable

### ISSUE THREAD

**Issue by NexoShez** [absolute-quantum/cats-blender-plugin#640]
I have the current version of Blender as of late Aug 2023 (or 3.3.10), & i cant enable the plugin; i can install it, but i cant enable it

error right here, please tell me how to fix this if anyone can ^^
![image](https://github.com/absolute-quantum/cats-blender-plugin/assets/98440899/3980346e-117c-46ef-a4b4-a3d8878b5421)

**Comment by JennaScvl** [comment]
I'm having the exact same problem. Blender 3.6

**Comment by PinkSpectrum** [comment]
Yeah same here is there any way to fix it? like going on a older version of blender or its an all system issue?

**Comment by unk35h** [comment]
> I have the current version of Blender as of late Aug 2023 (or 3.3.10), & i cant enable the plugin; i can install it, but i cant enable it
> 
> error right here, please tell me how to fix this if anyone can ^^ ![image](https://user-images.githubusercontent.com/98440899/263458670-3980346e-117c-46ef-a4b4-a3d8878b5421.png)


As the old issues mentioned, if you wanna use CATS in the Blender 3.5+, you have to download the development version (Code -> Download ZIP). Then in blender, you should choose the zip file for the installation. 
https://github.com/absolute-quantum/cats-blender-plugin/tree/development

**Comment by NexoShez** [comment]
> > I have the current version of Blender as of late Aug 2023 (or 3.3.10), & i cant enable the plugin; i can install it, but i cant enable it
> > error right here, please tell me how to fix this if anyone can ^^ ![image](https://user-images.githubusercontent.com/98440899/263458670-3980346e-117c-46ef-a4b4-a3d8878b5421.png)
> 
> As the old issues mentioned, if you wanna use CATS in the Blender 3.5+, you have to download the development version (Code -> Download ZIP). Then in blender, you should choose the zip file for the installation. https://github.com/absolute-quantum/cats-blender-plugin/tree/development

It works, thx ^^

**Comment by XYDIU** [comment]
Using the development version did not fix this issue for me, there is a solution in #631 that did solve the problem for me. (Using blender 3.6, and the development version of the plug-in.)

**Comment by Fuji-Modulation** [comment]
The solution written above didn't solve this issue for me. What did was a solution in #629 Simply need to delete one word from a line in the init.py file. More explained solution in other thread.

**Comment by nils-trubkin** [comment]
> The solution written above didn't solve this issue for me. What did was a solution in #629 Simply need to delete one word from a line in the init.py file. More explained solution in other thread.

Make sure it's the development branch you're getting, not master branch. Dev branch works fine.

**Comment by Urcreativemuse** [comment]
> > I have the current version of Blender as of late Aug 2023 (or 3.3.10), & i cant enable the plugin; i can install it, but i cant enable it
> > error right here, please tell me how to fix this if anyone can ^^ ![image](https://user-images.githubusercontent.com/98440899/263458670-3980346e-117c-46ef-a4b4-a3d8878b5421.png)
> 
> As the old issues mentioned, if you wanna use CATS in the Blender 3.5+, you have to download the development version (Code -> Download ZIP). Then in blender, you should choose the zip file for the installation. https://github.com/absolute-quantum/cats-blender-plugin/tree/development

i created an account just i can let people know it NEEDS to be devolvement cat and then it works just fine
